.vscode/launch.json

```
@@ -10,7 +10,7 @@
             "type": "cppdbg",
             "request": "launch",
             "program": "${workspaceFolder}/src/bin/lfortran",
-            "args": ["examples/expr2.f90"],
+            "args": ["--fixed-form","fmt","test2.f90"],
```

Now do `git pull <your-branch> <your-remote>` and then add a new commit:
```suggestion
            "args": ["examples/expr2.f90"],
```

---

@mohitdhingra22 go ahead and apply the above change.

---

All done or any changes to do?